DirectProgramming/DPC++FPGA/ReferenceDesigns/anr/src/column_stencil.hpp

```diff
@@ -4,13 +4,13 @@
 #include <CL/sycl.hpp>
 #include <sycl/ext/intel/fpga_extensions.hpp>
 
+#include "constexpr_math.hpp"
 #include "data_bundle.hpp"
-#include "mp_math.hpp"
 #include "shift_reg.hpp"
 #include "unrolled_loop.hpp"
 
 using namespace sycl;
-using namespace hldutils;
+using namespace fpga_tools;

```

Comment:
Do we want to use 'using' statements like this?  I thought this went against best coding practices?

You do this in other files as well, so if you decide to get rid of it here please grep for 'using namespace' in your code and fix it everywhere.